try {
    console.log(int('a'));
} catch (e) {
    console.log(e);
}
